.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.18.0
```

Comment:
While we're at it, would you mind adding a `.nvmrc` as well?

Reply:
Will do